documentation/pe_puppetdb_notes.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: "PuppetDB 3.2 » PE PuppetDB Notes"
+layout: default
+canonical: "/pe-puppetdb-notes"
+---
+
+[reports]: ./api/query/v4/reports.html
+[terminus]: ./puppetdb_connection.markdown#include_unchanged_resources
+
+## 2015.3.0
```

Comment:
I mean, do we want two release notes, PE notes already go into our release notes ... we can make them bold, put them in a section, whatever. Having two files means two files to update.

yeah ... I think a release note format is probably the wrong approach anyway for these kind of notes, expressing differences between FOSS and PE should be a new layout of some kind. If its required at all. I can see expressing large-set 'features' that require multiple configuration items to be enabled is not a bad thing.